.github/workflows/run-tests.yml

```diff
@@ -211,3 +228,14 @@ jobs:
             }
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+      - name: Clean up
+        id: docker-clean-up
+        if: ${{ env.ACT }}
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            docker compose -f tests/docker-compose.yml stop
+            docker compose -f tests/docker-compose.yml rm -f
```

Comment:
You want to possibly also remove the volumes. For github it doesn't matter, they won't persist anyway.

Reply:
Good idea to remove the volumes - they were stacking up and I'll put that change in the other actions too.

Turns out `-v` with `docker compose rm` only removes unnamed containers. With `docker compose down` -v removes named ones too. I experimented with the `--rmi local` flag too, but I still get dangling images left over. Adding `--rm` to the build command in the Makefile fixes this, (see https://github.com/UN-OCHA/cerf8/pull/1120), but I'm not sure if that interferes with anything else. Even if it is in the cause of making my life easier.